### PR TITLE
Wangdi/daos layout upgrade

### PR DESCRIPTION
**Pull request by wangdi1**

### CODE REVIEW

File: src/tests/ftest/daos_test/rebuild.yaml [hunk 1]
```diff
@@ -108,13 +108,8 @@ daos_tests:
     test_rebuild_28: -s3 -u subtests="28"
     test_rebuild_29: -s5 -u subtests="29"
   stopped_ranks:
-    test_rebuild_22: [7]
-    test_rebuild_23: [7]
-    test_rebuild_24: [7]
-    test_rebuild_25: [5, 6, 7]
     test_rebuild_26: ["random"]
     test_rebuild_27: ["random"]
-    test_rebuild_28: [7]
```
Comment (daltonbohning) [inline src/tests/ftest/daos_test/rebuild.yaml:117]: Is this because the test restarts the engines now?

Comment (wangdi1) [reply src/tests/ftest/daos_test/rebuild.yaml:117]: yes, the patch will restart these engines after shutdown for testing online OSA.

**Comment by ipoddubn** [comment]
please accelerate this PR, there's a request from a partner for this feature. thank you!

**Review by mjmac — COMMENTED** [review]
Go changes look fine to me, but they are a tiny portion of this PR so the +1s should come from other reviewers.

**Comment by wangdi1** [comment]
> please accelerate this PR, there's a request from a partner for this feature. thank you!

Sure, though this PR is just testing purpose, and the whole patch might include 5 or 6 smaller patches. I will try to accelerate the process. Thanks.

**Review by jolivier23 — COMMENTED** [review]
If this particular PR is just for  test, please make it a draft so it doesn't alert me every time  there are changes (due to teams)

**Comment by wangdi1** [comment]
> If this particular PR is just for test, please make it a draft so it doesn't alert me every time there are changes (due to teams)

Ah, sorry, I should close this PR.

**Pull request closed without merging**